src/client/gui/gui.cpp

```diff
@@ -526,6 +530,100 @@ void GUI::_layoutGameHUD() {
         ++it;
     }
     this->addWidget(std::move(durationFlex));
+
+    //  Display orb state here?
+
+    //  Orb state breakdown:
+    //  if match phase is MatchPhase::MazeExploration
+    //      --> orb hasn't been found yet
+    //  if match phase is MatchPhase::RelayRace
+    //      --> orb has been found at least once!
+    //          if the orb is currently being carried, print which player name
+    //              is carrying it
+    //          otherwise, "the orb is lost"
+    std::string orbStateString;
+    if (client->gameState.matchPhase == MatchPhase::MazeExploration) {
+        orbStateString = "The Orb is hidden somewhere in the Labyrinth...";
+    }
+    else {
+        bool orbIsCarried = false;
+        for (auto it : client->gameState.lobby.players) {
```

Comment:
super minor but you could use tuple destructuring here to make it a tiny bit more readable, so it would be like

```cpp
for (auto [id, name] : client->gameState.lobby.players
```